content/applications/sales/sales/send_quotations/get_signature_to_validate.rst

```diff
@@ -2,51 +2,76 @@
-   :guilabel:`Other Info` tab.
+   On standard quotations, this option is the :guilabel:`Signature` feature, located under the
+   :guilabel:`Other Info` tab of the quotation form.
 
    .. image:: get_signature_to_validate/signature-other-info-tab.png
       :align: center
-      :alt: How to enable online signature on standard quotations.
+      :alt: The online signature feature option in the Other Info tab of a quotation form in Odoo.
 
 Order confirmations with online signatures
 ==========================================
 
-When quotations are sent to clients, there's an option for them to :guilabel:`Sign & Pay` directly
-on the quotation in the online customer portal. When clicked, clients have the option to draw a
-signature, fill in the field with an automated signature, or load a file from their computer.
+When clients access quotations in their customer portal, there's a :guilabel:`Sign & Pay` button
+directly on the quotation.
+
+.. image:: get_signature_to_validate/sign-and-pay-button.png
+   :align: center
+   :alt: The Sign and Pay button present on online quotations in Odoo Sales.
+
+When clicked, a :guilabel:`Validate Order` pop-up window appears. In this pop-up window, the
+:guilabel:`Full Name` field is auto-populated, based on the contact information in the database.
 
-.. image:: get_signature_to_validate/signature-validate-order.png
+.. image:: get_signature_to_validate/validate-order-popup.png
    :align: center
-   :alt: How to confirm an order with a signature on Odoo Sales.
+   :alt: The Validate Order pop-up window for online signatures in Odoo Sales.
+
+Then, customers have the option to enter an online signature with any of the following options:
+:guilabel:`Auto`, :guilabel:`Draw`, or :guilabel:`Load`.
+
+:guilabel:`Auto` lets Odoo automatically generate an online signature based on the information in
+the :guilabel:`Full Name` field. :guilabel:`Draw` lets the customer use the cursor to create a
+custom signature directly on the pop-up window. And :guilabel:`Load` lets the customer upload a
+previously-created signature file from their computer.
+
+When they have chosen any of the three previously mentioned signature options (:guilabel:`Auto`,
+:guilabel:`Draw`, or :guilabel:`Load`), they will click the :guilabel:`Accept & Sign` button.
+
+When :guilabel:`Accept & Sign` is clicked, the various payment method options become available for
+them to choose from.
 
-Once signed, the various payment method options become available. Then, when the quotation is paid
-and confirmed, a delivery order is automatically created (if the Odoo *Inventory* app is installed).
+Then, when the quotation is paid and confirmed, a delivery order is automatically created (if the
+Odoo *Inventory* app is installed).
 
 .. seealso::
-   :doc:`/applications/sales/sales/send_quotations/quote_template`
+   :doc:`/applications/sales/sales/send_quotations/quote_template` &
+   :doc:`/applications/sales/sales/send_quotations/get_paid_to_validate`
```

Comment:
Let's put these doc links in separate paragraphs instead of having them on one line separated by "&"
```suggestion
   :doc:`/applications/sales/sales/send_quotations/quote_template`
   :doc:`/applications/sales/sales/send_quotations/get_paid_to_validate`
```